src/libraries/System.Private.CoreLib/src/System/TimeSpan.cs

```diff
@@ -271,6 +271,22 @@ public static TimeSpan FromDays(double value)
             return Interval(value, TicksPerDay);
         }
 
+        public static TimeSpan FromDays(int days)
```

Comment:
>public [](http://example.com/codeflow?start=8&length=6)

All APIs need to have the Xml doc elements.

Reply:
Haha I originally wrote it with the XML Docs but noticed that the older methods do not have any documentation and removed them. No problem.

Reply:
Yes we are slowly adding missing XML docs through these libraries. Historically docs were not generated from sources (unlike most .NET libraries out there - perhaps because many were written before the XML generates useful docs). As we copy the official docs onto the API (aided by tools) we switch one library after another to being the "source of truth" for docs and eventually they all will be. There is an ongoing project for this in case you are interested in contributing to it at some point.